core-plugins/src/test/java/co/cask/hydrator/plugin/batch/BatchPluginsTestSuite.java

```diff
@@ -25,7 +26,6 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-  BatchCubeSinkTestRun.class,
```

Comment:
Are you sure the test runs? I think we don't run tests whose classes end with Run.